README.md

```diff
@@ -16,7 +16,7 @@
 Add your answers to this markdown file.
 
 
-1. One reason asymptotic analysis may be misleading is that we ignore constant factors within the algorithms, but constant factors can have a large effect on the overall runtime. For example, a function could have a linear time complexity, but a function that has runtime 1(n) vs 1000(n) could be a significant difference in runtime, yet bot functions have the same time complexity. Another reason asymptotic analysis may be misleading is because the bounds are very loose, so the overall time complexity is not very accurate in runtime prediction. Finally, a third reason why asymptotic analysis can be misleading is that it ignores hardware factors. A computer that has a computer with a lot of computational power will perform significantly faster with the same algorithm as a computer that has very poor computational power. While these hardware factors are constant factors, they can have a large affect on the time it takes for a function to run. 
+1. One reason asymptotic analysis may be misleading is that we ignore constant factors within the algorithms, but constant factors can have a large effect on the overall runtime. For example, a function could have a linear time complexity, but a function that has runtime 1(n) vs 1000(n) could be a significant difference in runtime, yet bot functions have the same time complexity. Another reason asymptotic analysis may be misleading is because the bounds are very loose, so the overall time complexity is not very accurate in runtime prediction. Finally, a third reason why asymptotic analysis can be misleading is that it ignores hardware factors. A computer that has a computer with a lot of computational power will perform significantly faster with the same algorithm as a computer that has very poor computational power. Hardware factors could be constant or exponential factors that significantly slow down the runtime of a function. For example, if the computer's RAM does not have enough memory to hold the data in the function, it may have to switch to the hard drive to store the data. This can exponentially affect the runtime of a function because the speed at which the RAM retrieves the data may be exponentially faster than the speed at which the hard drive retrieves data.
```

Comment:
But we can analyze memory complexity, which would give us a hint whether a program might run out of physical memory.